wpimath/src/main/native/include/frc/apriltag/ApriltagFieldLayout.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <cstdint>
+#include <string_view>
+#include <vector>
+
+#include <wpi/SymbolExports.h>
+
+#include "frc/geometry/Pose2d.h"
+#include "frc/geometry/Pose3d.h"
+#include "frc/geometry/Transform2d.h"
+#include "units/acceleration.h"
+#include "units/curvature.h"
+#include "units/time.h"
+#include "units/velocity.h"
+
+namespace wpi {
+class json;
+}  // namespace wpi
+
+namespace frc {
+class WPILIB_DLLEXPORT ApriltagFieldLayout {
+ public:
+    struct WPILIB_DLLEXPORT Apriltag {
+        int id;
+
+        Pose3d pose = Pose3d();
+    };
+
+    ApriltagFieldLayout() = default;
+    
+    explicit ApriltagFieldLayout(const std::vector<Apriltag>& apriltags);
+    
+    const std::vector<Apriltag>& GetTags() const { return m_apriltags; };
+    
+    Pose3d GetTagPose(int id) const;
+    
+    void setShouldMirror(bool mirror) { m_mirror = mirror; }
```

Comment:
The first letter of member function names should be capitalized in C++. Their implementations should be in a .cpp file instead of the .h.